cypress/integration/features/course/deleteCourse.feature

```diff
@@ -2,8 +2,8 @@ Feature: To delete the course created for testing purpose
 
     As a teacher I want to delete the newly created test course so that list of course can be cleaned and not full with test courses
 
-    Scenario: Deleting the test course created during executing the testing
-        Given I am logged in as a 'teacher' at 'brb'
+    Scenario: Deleting the course/room which is created after executing the create course feature
```

Comment:
Is the course/room created by executing the create course feature or is it created after that feature?